packages/odyssey-react-mui/src/themes/odyssey/shape.ts

```diff
@@ -13,6 +13,12 @@
 import type { ThemeOptions } from "@mui/material";
 import * as Tokens from "@okta/odyssey-design-tokens";
 
+// Strip units from BorderRadiusBase to accommodate MUI's typing
+const StrippedBorderRadiusBase =
+  typeof Tokens.BorderRadiusBase === "string"
+    ? Tokens.BorderRadiusBase.replace(/\D/g, "")
```

Comment:
MY BAD! The proper regex is `/(\d+).*/`. That's why it didn't work before.

![image](https://user-images.githubusercontent.com/97464104/195197554-170dc4b9-3a37-47aa-adb3-1358fa2f7d52.png)